version/version.go

```diff
@@ -1,5 +1,3 @@
 package version
 
-var (
-	Version = "0.0.1"
-)
+var Version = "0.0.1"
```

Comment:
What do we even need this for? If at all, shouldn't this be set during build time, and not hardcoded?

Reply:
The `version` pkg originates from the "plain" project bootstrapped by `operator-sdk`, and is used in `pkg/manager/main.go`'s  `printVersion()`. I don't see any use including it at all, hence i'm dropping it for now.